### DISCUSSION TITLE
Downloader struggles with long timelines

### DISCUSSION THREAD

**Discussion by Wankokili** [Avnsx/fansly#151] [Q&A] [answered]
Hello!
I tried to use the downloader, it works quite well but one issue i've noticed is how on some accounts's timelines, scrapped content is limited to a few months before present day. especially is the account is posting very frequently.

**Comment by Sebastian1175** [comment]
It seems like the Downloader has in general some issues atm. It download one Timeline part and then the next they cant scrape anything.

**Comment by Avnsx** [comment, marked as answer]
duplicate of https://github.com/Avnsx/fansly-downloader/issues/148
